### PR TITLE
make tlsconfig enabled not required

### PR DESCRIPTION
**Pull request by ImNumber4**
### What changed? Why?
make tlsconfig enabled not required otherwise there is no way to turn it off
 
### How did you test the change?

- [x] unit test
- [ ] integration test
- [ ] functional test
- [ ] adhoc test (described below)

### CODE REVIEW

**Review by leozc — APPROVED** [review]
approve

**Review by HenryYYang — DISMISSED** [review]

File: config/chainstorage/ethereum/mainnet/development.yml [hunk 1]
```diff
@@ -1,40 +1,45 @@
 # This file is generated by "make config". DO NOT EDIT.
 aws:
-  aws_account: development
-  bucket: example-chainstorage-ethereum-mainnet-dev
+  aws_account: 281895244027
```
Comment (HenryYYang) [inline config/chainstorage/ethereum/mainnet/development.yml:3]: this file shouldn't be part of this change.

**Review by HenryYYang — CHANGES_REQUESTED** [review]

**Comment by cb-heimdall** [comment]
Review Error for HenryYYang @ 2023-12-30 07:29:39 UTC
User failed mfa authentication, public email is not set on your github profile. see [go/mfa-help](http://go/mfa-help)

**Review by HenryYYang — APPROVED** [review]
